src/assets/icons/play.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
kill this, it's just like `ChevronRight` but without the proper viewbox spacing

Reply:
Kept this because it's still 1:1 ratio. Don't want to bother design to export a new svg at the correct 24/24 since it would look the same anyway.